lib/rucio/core/transfer.py

```diff
@@ -671,8 +671,8 @@ def __rewrite_source_url(source_url, source_sign_url, dest_sign_url, source_sche
         source_url = re.sub('davs', 's3s', source_url)
         source_url = re.sub('https', 's3s', source_url)
```

Comment:
I believe all these previous lines potentially have the same issue. Why don't we fix them too ?